rdmd.d

```diff
@@ -15,9 +15,10 @@
 // Written in the D programming language.
 
 import std.algorithm, std.array, core.stdc.stdlib, std.datetime,
-    std.digest.md, std.exception, std.file, std.getopt,
+    std.digest.md, std.exception, std.getopt,
     std.parallelism, std.path, std.process, std.range, std.regex,
     std.stdio, std.string, std.typetuple;
+static import std.file;
```

Comment:
Note that an alternative to `static import std.file` could be to use _selective_ imports that very carefully determine which `std.file` functions are allowed to be used unwrapped.

Reply:
yes I'm aware.  I think static in this case makes more sense.